packaging_automation/tests/files/citus.spec

```diff
@@ -81,6 +84,66 @@ echo %{pginstdir}/share/extension/%{sname}.control >> installation_files.list
 %doc %{pginstdir}/doc/extension/NOTICE-%{sname}
 
 %changelog
+* Tue Feb 01 2022 - Gurkan Indibay <gindibay@microsoft.com> 10.1.4.citus-1
+- Official 10.1.4 release of Citus
+
+* Mon Nov 29 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.2.3.citus-1
+- Official 10.2.3 release of Citus
+
+* Fri Nov 12 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.0.6.citus-1
+- Official 10.0.6 release of Citus
+
+* Mon Nov 08 2021 - Gurkan Indibay <gindibay@microsoft.com> 9.5.10.citus-1
+- Official 9.5.10 release of Citus
+
+* Thu Nov 04 2021 - Gurkan Indibay <gindibay@microsoft.com> 9.2.8.citus-1
+- Official 9.2.8 release of Citus
+
+* Wed Nov 03 2021 - Gurkan Indibay <gindibay@microsoft.com> 9.2.7.citus-1
+- Official 9.2.7 release of Citus
+
+* Thu Oct 14 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.2.2.citus-1
+- Official 10.2.2 release of Citus
+
+* Fri Sep 24 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.2.1.citus-1
+- Official 10.2.1 release of Citus
+
+* Fri Sep 17 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.1.3.citus-1
+- Official 10.1.3 release of Citus
+
+* Thu Sep 16 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.2.0.citus-1
+- Official 10.2.0 release of Citus
+
+* Tue Aug 17 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.1.2.citus-1
+- Official 10.1.2 release of Citus
+
+* Tue Aug 17 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.0.5.citus-1
+- Official 10.0.5 release of Citus
+
+* Tue Aug 17 2021 - Gurkan Indibay <gindibay@microsoft.com> 9.5.7.citus-1
+- Official 9.5.7 release of Citus
+
+* Wed Aug 11 2021 - Gurkan Indibay <gindibay@microsoft.com> 9.4.6.citus-1
+- Official 9.4.6 release of Citus
+
+* Fri Aug 06 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.1.1.citus-1
+- Official 10.1.1 release of Citus
+
+* Fri Jul 16 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.1.0.citus-1
+- Official 10.1.0 release of Citus
+
+* Fri Jul 16 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.0.4.citus-1
+- Official 10.0.4 release of Citus
+
+* Fri Jul 09 2021 - Gurkan <gindibay@microsoft.com> 9.5.6.citus-1
+- Official 9.5.6 release of Citus
+
+* Thu Jul 08 2021 - Gurkan <gindibay@microsoft.com> 9.4.5.citus-1
+- Official 9.4.5 release of Citus
+
+* Thu Mar 18 2021 - Gurkan Indibay <gindibay@microsoft.com> 10.0.3.citus-1
+- Official 10.0.3 release of Citus
+
```

Comment:
Do we really need to have so many rows in the test file?

Does it not make sense to test the behaviour on a simpler and shorter spec?